biocrnpyler/species.py

```diff
@@ -222,6 +243,7 @@ def __eq__(self, other):
                             and self.name == other.name \
                             and set(self.attributes) == set(other.attributes)\
                             and self.parent == other.parent\
+                            and self.compartment == other.compartment\
                             and self.position == other.position:
```

Comment:
I want to confirm that this equality test is universal and will work with all Species [I'm worried that it might have been overwritten in something like OrderedPolymerSpecies. Just making sure we do the compartment check everywhere we need.

Reply:
I double checked and I think this change is made everywhere in species.py. If you find some place that I might have missed please let me know.